modules/ROOT/pages/traversal-framework/traversal_framework_example.adoc

```diff
@@ -0,0 +1,220 @@
+:description: A Neo4j Traversal Framework example.
+
+:org-neo4j-graphdb-Direction-both: {neo4j-javadocs-base-uri}/org/neo4j/graphdb/Direction.html#BOTH
+
+[[examples-how-to-use-the-traversal-framework]]
+= Traversal Framework Example
+
+Here are some examples to highlight how one might use the Traversal Framework.
+See the graph below which illustrates a small group of friends.
+
+image::traversal_framework_example.svg[role="middle"]
```

Comment:
While doing an editorial review, I realized this image is missing. Could you please check it @nadja-muller @gem-neo4j ?

Reply:
Nice finds, I have added a new PR based on these 2 (as this branch was already merged) :) https://github.com/neo4j/docs-java-reference/pull/40